app/components/daisy/actions/dropdown_component.rb

```diff
@@ -16,6 +16,8 @@
 #   and is styled automatically.
 # @slot activator A custom (i.e. non-button) activator for the dropdown.
 #   Automatically adds the `role="button"` and `tabindex="0"` attributes.
+# @slot input A text input field that can be included in the dropdown menu, useful
```

Comment:
These docs are wrong and the whole slot can probably be removed... or actually do what it says and put the input in the dropdown menu